content/changelog-stable/index.html.haml

```diff
@@ -3,40 +3,30 @@ layout: changelog
 title: LTS Changelog
 show_ratings: true
 has_rss: true
+actions:
+  <a class="app-button app-button--primary app-mobile-hide" href="/doc/upgrade-guide">
+    <ion-icon name="arrow-up-circle-outline"></ion-icon>
+    Upgrade Guide
+  </a>
 ---
 
-%div{:class => 'app-banner'}
-  See the
-  %a{:href => '/doc/upgrade-guide/'}
-    LTS upgrade guide
-  for advice on upgrading Jenkins.
-
-%div{:class => 'app-banner'}
-  See the
-  %a{:href => '/download/lts'}
-    LTS Release Line
-  for more information on Jenkins LTS.
-
 .ratings
   - # source: https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/lts.yml
   - site.changelogs[:lts].reverse_each do | release |
     - if Gem::Version.new(release.version) <= Gem::Version.new(site.jenkins.stable)
-      = partial('release-header.html.haml', :release => release)
-      %p
-        %small
-          %a{:href => "/changelog-stable/#{release.version}/" }
-            permalink to this entry
-      -if release.changes and release.lts_changes and release.lts_baseline
-        %div
-          %strong
+      %div
+        = partial('release-header.html.haml', :release => release, :url => 'changelog-stable')
+        - if release.changes and release.lts_changes and release.lts_baseline
+          .app-releases__notable-changes{:style => "margin-top: 15px"}
             -# TODO figure out how to get rid of this
-            = "Changes since #{release.lts_baseline}:"
-      %ul.image
-        = partial('changelog-changes.html.haml', :changes => release.changes)
-      -if release.changes and release.lts_changes
-        %div
-          %strong
-            = "Notable changes since #{release.lts_predecessor}:"
-        %ul.image
-          = partial('changelog-changes.html.haml', :changes => release.lts_changes)
+            = "Changes since #{release.lts_baseline}"
+        %div.app-releases__list__items
+          = partial('changelog-changes.html.haml', :changes => release.changes)
+
+    - if release.changes and release.lts_changes
+      .app-releases__notable-changes
+        = "Notable changes since #{release.lts_predecessor}"
+      %div.app-releases__list__items
+        = partial('changelog-changes.html.haml', :changes => release.lts_changes)
```

Comment:
This is not indented far enough, so https://www.jenkins.io/changelog-stable/ currently has "Notable changes" for an unreleased version of Jenkins.

Reply:
https://github.com/jenkins-infra/jenkins.io/pull/7636